javascript/javascript1/week1/pizza-exercise.js

```diff
@@ -0,0 +1,16 @@
+console.log("I love pizza");
+let pizzaName = "chicken-pizza";
+let pizzaPrice =10;
+let noOfPizza = 2;
+let familySize = true;
+let totalPrice;
+
+if (familySize) {
+    totalPrice= pizzaPrice*2*noOfPizza;
+}
+
```

Comment:
Avoid inserting empty lines between the `if` part and the `else` part because this visually separates them even though they "belong" together.

Reply:
I will always remember. done now